examples/vello_editor/src/main.rs

```diff
@@ -340,6 +358,7 @@ fn main() -> Result<()> {
         scene: Scene::new(),
         editor: text::Editor::new(text::LOREM),
         last_drawn_generation: Default::default(),
+        last_sent_ime_cursor_area: None,
```

Comment:
Maybe this should be an arbitrary definitely invalid rect? E.g. `0x0 at (0, 0)` or similar

Reply:
It's now flattened and splatted with NaN, which should definitely never be returned.